website/docs/docs/dbt-versions/core-upgrade/06-upgrading-to-v1.9.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Upgrading to v1.9 (beta)"
+id: upgrading-to-v1.9
+description: New features and changes in dbt Core v1.9
+displayed_sidebar: "docs"
+---
+ 
```

Comment:
do we add any sort of callout that this is "beta" - or is the title clear enough?

Reply:
@graciegoheen The version drop down will also show beta. I think this will be fine and is what we've done previously but open to adding a note. Let me know if you feel strongly pro note! Here's how it [looks now](https://docs-getdbt-com-git-upgrade-19-guide-dbt-labs.vercel.app/docs/dbt-versions/core-upgrade/upgrading-to-v1.9).

Reply:
If this is what we've done before, that's fine with me!